public/electron.js

```diff
@@ -34,7 +34,7 @@ if (process.platform === 'darwin') {
 }
 
 const appName = 'Hathor Wallet';
-const walletVersion = '0.18.0';
+const walletVersion = '0.18.1';
```

Comment:
I feel we shouldn't change the version here.

Reply:
You mean shouldn't change in this file or shouldn't change in this PR?

Reply:
I think we shouldn't change in this PR. It might be part of other changes. I think it is better to create a separate PR to increase the version, and we can take the opportunity to put the changelog in that PR.

Reply:
Ok, makes sense. Will rollback those changes